airbyte-integrations/connectors/source-mssql/src/test-integration/java/io/airbyte/integrations/source/mssql/MssqlSourceAcceptanceTestSSL.java

```diff
@@ -0,0 +1,114 @@
+package io.airbyte.integrations.source.mssql;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.node.ObjectNode;
+import com.google.common.collect.ImmutableMap;
+import io.airbyte.commons.json.Jsons;
+import io.airbyte.commons.resources.MoreResources;
+import io.airbyte.db.Database;
+import io.airbyte.db.Databases;
+import io.airbyte.integrations.standardtest.source.SourceAcceptanceTest;
+import io.airbyte.protocol.models.CatalogHelpers;
+import io.airbyte.protocol.models.ConfiguredAirbyteCatalog;
+import io.airbyte.protocol.models.ConnectorSpecification;
+import io.airbyte.protocol.models.Field;
+import org.apache.commons.lang3.RandomStringUtils;
+import org.testcontainers.containers.MSSQLServerContainer;
+import org.testcontainers.utility.DockerImageName;
+
+import java.sql.SQLException;
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.List;
+
+public class MssqlSourceAcceptanceTestSSL extends SourceAcceptanceTest {
```

Comment:
@heade please rename MssqlSourceAcceptanceTestSSL ->SslEnabledMssqlSourceAcceptanceTest

Reply:
@yaroslav-hrytsaienko the name is similar to the destination-mssql tests
but I like this option too